src/app/data-explorer/edit-request-dialog/edit-request-dialog.component.spec.ts

```diff
@@ -0,0 +1,209 @@
+/*
+Copyright 2022-2023 University of Oxford
+and Health and Social Care Information Centre, also known as NHS Digital
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+import { Validators } from '@angular/forms';
+import {
+  MatDialog,
+  MatDialogActions,
+  MatDialogContent,
+  MatDialogRef,
+  MAT_DIALOG_DATA,
+} from '@angular/material/dialog';
+import { MatFormField, MatLabel } from '@angular/material/form-field';
+import { MockComponent } from 'ng-mocks';
+import { Observable, of } from 'rxjs';
+import { dontAllowDuplicatedNames } from 'src/app/shared/mdm-validators';
+import { createDataRequestsServiceStub } from 'src/app/testing/stubs/data-requests.stub';
+import {
+  createMatDialogRefStub,
+  createMatDialogStub,
+} from 'src/app/testing/stubs/mat-dialog.stub';
+import {
+  ComponentHarness,
+  setupTestModuleForComponent,
+} from 'src/app/testing/testing.helpers';
+import { DataRequestsService } from '../data-requests.service';
+import { OkCancelDialogResponse } from '../ok-cancel-dialog/ok-cancel-dialog.component';
+
+import {
+  EditRequestDialogComponent,
+  EditRequestDialogOptions,
+  EditRequestDialogResponse,
+} from './edit-request-dialog.component';
+
+describe('CreateRequestDialogComponent', () => {
```

Comment:
Apologies, I should have spotted this on my first review. This test is badly named. It is not the CreateRequestDialogComponent which caused me confusion when this test started to fail.

Reply:
Oh, sorry about that! Last commit fixes these issues.